front/app/containers/Admin/projects/all/Lists/ModeratorProjectList.tsx

```diff
@@ -61,13 +42,33 @@ const ModeratorProjectList = memo(() => {
               return (
                 !isProjectFoldersEnabled ||
```

Comment:
Minor, we could return null if the condition is not met. This being a map with a return, that condition will always return something.

Reply:
I've improved these overall because the current checks didn't make sense. We only want this check for the folder row. 

A lot about folders was implemented poorly so we need to be on the lookout. Thanks for catching. :-)